apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -865,6 +942,10 @@ int itc_uart_main(void)
 	itc_systemio_iotbus_uart_set_mode_write_read_p();
```

Comment:
Could you please add a "TODO" comment for line 939
`//itc_systemio_iotbus_uart_write_read_flush_read_p();// TASH being hanged on execution of the testcase`
If this particular testcase is not resolved and is left to be resolved later.